runelite-client/src/main/java/net/runelite/client/util/StackFormatter.java

```diff
@@ -152,6 +160,24 @@ else if (quantity < 10_000_000)
 	 * with K after 100,000 and M after 10,000,000
 	 */
 	public static String quantityToRSDecimalStack(int quantity)
+	{
+		return quantityToRSDecimalStack(quantity, false);
```

Comment:
Is there anything calling this that doesn't want the more precise values? Just looks like the ge offer panel and the ammo counter.

Reply:
Well, the point of the RS decimal stack method is to format numbers exactly like OSRS formats amounts, so I think having separate method for this is good.